index.js

```diff
@@ -138,7 +139,9 @@ module.exports = {
     "require-yield": 1,
     "space-infix-ops": 1,
     "space-in-parens": [1, "never"],
-
+    "space-before-blocks": 1,
+    "func-style": [1, "declaration", { "allowArrowFunctions": true }],
```

Comment:
Hm, we do use that style:

```js
const foo = () => {}
```

so I guess we should use the `expression` option.